lib/hacluster.pm

```diff
@@ -73,6 +73,22 @@ our $prev_console;
 our $join_timeout    = bmwqemu::scale_timeout(60);
 our $default_timeout = bmwqemu::scale_timeout(30);
 
+# Private functions
+sub _just_the_ip {
+    my $node_ip = shift;
+    if ($node_ip =~ /(\d+\.\d+\.\d+\.\d+)/) {
+        return $1;
+    }
+    return 0;
+}
+
+sub _test_return_var {
+    my $var = shift;
+
+    die 'Last command did not return a defined value!' if !defined $var;
```

Comment:
I think method name can be changed to `_test_return_var_defined` or to `_test_var_defined` so it is more clear what the method is doing.

You can also use `unless defined $var` instead of `if !defined $var`.

You can even do:
```
sub _test_var_defined {
    die 'Last command did not return a defined value!' unless defined shift;
}
```
One-liner FTW :P

Reply:
Yes, your suggested name is better, I will change this.
As for the code, I simply did a copy/paste of a line, even if I know `unless`... thx!

Reply:
Done!